### PR TITLE
added my metadata and also added a robot arm post on projects

### PR DESCRIPTION
**Pull request by raghavauppuluri13**
Added author info, and a robot arm post update. Will add some news updates about it in a future pull request

### CODE REVIEW

File: _data/authors.yml [hunk 1]
```diff
@@ -7,3 +7,9 @@ jamesb:
 hmccarty:
   name: Harrison McCarty
   web: mailto:mccarth@purdue.edu
+
+ruppuluri:
+  name: Raghava Uppuluri
+  avatar: members/raghava.jpg
```
Comment (jrbaxter0) [inline _data/authors.yml:13]: png is preferred, but I don't think it matters too much as long as it displays nicely

**Review by jrbaxter0 — COMMENTED** [review]
Assuming it all renders nicely, it looks good. I'll check that out later today if I can.

File: _data/authors.yml (hunk 1 above)
Comment (raghavauppuluri13) [reply _data/authors.yml:13]: Yeah, I checked and it renders nicely.

**Comment by jrbaxter0** [comment]
@raghavauppuluri13, so it doesn't seem to render exactly how I'd like. 

It would be nice if all the member profiles were the same size (either through better formatting of the page, or editing the picture you provided. I edited them all in the past, which maybe wasn't the best way)

image

And again, it would be nice to have all these match.
image

These aren't super critical, but I think it makes the site look more polished

**Comment by raghavauppuluri13** [comment]
Ah I see. I can change the resolutions for the project image, but is there a way to set a fixed size for the pfps?

**Comment by jrbaxter0** [comment]
> 
> 
> Ah I see. I can change the resolutions for the project image, but is there a way to set a fixed size for the pfps?

I have no idea. I've tried modifying the template in the past, but webdev is not my strong suit.

**Comment by hmcty** [comment]
I added auto-sizing for member and project features, check it out [here](https://github.com/purdue-arc/purdue-arc.github.io/pull/14)

**Comment by hmcty** [comment]
Resolved the merge conflict, can we merge this in as well?

**Comment by raghavauppuluri13** [comment]
@jrbaxter0 since there is auto image sizing. can this be merged?

**Pull request merged**